lib/tasks/mailman.rake

```diff
@@ -2,10 +2,7 @@ desc 'Send email reminder about upcoming checkins'
 task send_upcoming_checkin_reminder: :environment do
   if AppConfig.first.upcoming_checkin_email_active?
     # get all reservations that end today and aren't already checked in
-    upcoming_reservations =
-      Reservation.where('checked_out IS NOT NULL and checked_in IS NULL and '\
-                          'due_date >= ? and due_date < ?',
-                        Time.zone.today, Time.zone.today + 1.day)
+    upcoming_reservations = Reservation.due_soon
```

Comment:
@esoterik here's the issue - we only added the date condition to the `due_soon` scope, but not the `checked_out` condition. It's probably just this scope then, I'll merge #1322 back into #1288.